salt/modules/virt.py

```diff
@@ -1759,6 +1783,11 @@ def update(
 
         .. versionadded:: 3000
 
+    :param serials: list of serial connections
+        .. versionadded:: Sodium
+    :param consoles: list of console parameters
+        .. versionadded:: Sodium
```

Comment:
Refer to the documentation of the properties of items in the list contained in `init()`. See how this is done for disks, network interfaces and graphics already.